core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -946,6 +946,10 @@ public FileInfo getFileInfo(long fileId)
   @Override
   public FileInfo getFileInfo(AlluxioURI path, GetStatusContext context)
       throws FileDoesNotExistException, InvalidPathException, AccessControlException, IOException {
+    if (context.getOptions().hasDirectUfsAccess() && context.getOptions().getDirectUfsAccess()) {
```

Comment:
On 2.x, there is still no link from the master to the ufs request. Does this break the current architecture? Regardless of worker or proxy, they all get the mount id and attributes from the master and request ufs independently.